backend/src/cmd/import_realm_tree.rs

```diff
@@ -266,8 +291,7 @@ impl Block {
                             .query("select id from series where opencast_id = $1", &[uuid])
                             .await?;
                         if rows.is_empty() {
-                            warn!("Series with UUID '{}' not found! Skipping.", uuid);
-                            return Ok(());
+                            anyhow::bail!("Series with UUID '{}' not found! Skipping.", uuid);
```

Comment:
So this means that if a series can't be found, the whole process fails with an error? Is this desired?

Reply:
Yes. Tho I see that it's not ideal to only throw an error at that point. Ideally we should quickly check whether all series are present that are referenced in the file at the beginning. Note that all series that are added by the dummy blocks function should exist or otherwise it's a bug in that process. So yeah, if a series in the input file does not exist, we would like the user to know. However, the "Skipping" in that error message is wrong of course. Will remove.